src/test/java/org/cactoos/io/WriterAsOutputStreamTest.java

```diff
@@ -57,35 +58,31 @@
 
     @Test
     public void writesToByteArray() {
-        final ByteArrayOutputStream baos = new ByteArrayOutputStream();
         final String content = "Hello, товарищ! How are you?";
-        MatcherAssert.assertThat(
+        final ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        new Assertion<>(
             "Can't copy Input to Writer",
-            new TextOf(
-                new TeeInput(
-                    new InputOf(content),
-                    new OutputTo(
-                        new WriterAsOutputStream(
-                            new OutputStreamWriter(
-                                baos, StandardCharsets.UTF_8
-                            ),
-                            StandardCharsets.UTF_8,
-                            // @checkstyle MagicNumber (1 line)
-                            13
+            () -> new TextOf(
+                new Sticky(
```

Comment:
@Iprogrammerr Is necessary use the Sticky decorator here?

Reply:
@fabriciofx Same as above.

Reply:
@Iprogrammerr the same above.